sdk/core/core-rest-pipeline/src/fetchHttpClient.ts

```diff
@@ -252,6 +255,7 @@ function buildBodyStream(
       new TransformStream({
         transform(chunk, controller) {
           if (chunk === null) {
+            onEnd?.();
```

Comment:
should this live in the `flush()` handler instead?

Reply:
Yes, moved to `flush()` handler